docs/local_tutorial.md

```diff
@@ -169,6 +169,24 @@ kubectl get solrclouds -w
 # Hit Control-C when done
 ```
 
+The SolrCloud CRD is setup so that it is able to run with the HPA.
```

Comment:
let's also mention here that `spec.replicas` should be left unset in order to avoid overwrite when  `kubectl apply` happens